### DISCUSSION TITLE
Registration with inputimages with different qform in header

### DISCUSSION THREAD

**Discussion by MRIDude** [ANTsX/ANTs#1768] [General]
Just recently, I generated a template using antsMultivariateTemplateConstruction2.sh. The template was generated with both T1 and T2 from a large cohort. I inadvertently gave the scripts T1s and T2s that were not co-registered.  Surprisingly, the templates ended up being perfect. I found no issue with them.  When I load the T1/T2 in to fsleyes, they were co-registered which makes me think that the sform/qform contained the necessary info to align them.   Is this how antsRegistration works?

**Comment by cookpa** [comment]
Registration works in the physical space, so each T1w should be in the same physical space as its T2w. If you load a pair into ITK-SNAP, they should be aligned. If they are, the template script should work as expected.

**Reply by MRIDude** [reply]
Out of curiosity. Is there a tool to get an image in the same physical as another without resampling the images assuming the transform is rigid body?

**Reply by gdevenyi** [reply]
You would need to manipulate the qform/sform info, FSL tools are the best bet here, but no idea exactly how to do it.
